### DISCUSSION TITLE
incompatible version

### DISCUSSION THREAD

**Discussion by mryetin** [lloesche/valheim-server-docker#155] [Q&A] [answered]
Dear Valheim Player,

Can somebody help my with updating my server on Synology NAS?
I stopped the server, cleared it and downloaded the lastest version.. But i get still the same error after i enter my password ingame 'incompatible version'. Please help my because after each update i have this same problem. 

When i try to connect localy and not on my server its works. 

Thinks I've tried in the meantime..
Started Putty and login in the server
sudo docker exec -it valheim-server /bin/bash // open container
cd opt/steamcmd/ // to the steamcmd directory
./steamcmd.sh +login anonymous +force_install_dir ../valheim/server +app_update 896660 validate +exit // updating server
rebooting the server

But still the same problem :(

Some log files from the server:
03/03/2021 21:04:38: Got connection SteamID (ID)
03/03/2021 21:04:38: Got handshake from client (ID)
03/03/2021 21:04:41: VERSION check their:0.147.3  mine:0.147.3@0.9
03/03/2021 21:04:41: Peer (ID) has incompatible version, mine:0.147.3@0.9 remote 0.147.3
03/03/2021 21:04:41: Closing socket (ID)

FIX:

You need also install Valheim Plus (MOD) on Client side:
https://survivalservers.com/wiki/index.php?title=How_to_Install_Valheim_Plus

**Comment by lloesche** [comment]
@mryetin you have VALHEIM_PLUS turned on on the server but not installed on the client.

**Reply by mryetin** [reply]
I fixed it with the answer below, thanks for your reply

**Comment by anpross** [comment, marked as answer]
in case, like me, you only want to use server-features, please check the sample config in the Readme.md valhaim-plus section.

**Reply by lloesche** [reply]
Right, so if you don't need the client side features you can turn off the version check via `-e VPCFG_Server_enabled=true -e VPCFG_Server_enforceMod=false` plus whatever else options you're interested in. See https://github.com/lloesche/valheim-server-docker#configuration for details.

**Reply by mryetin** [reply]
Why I need or want client features on my server?

**Reply by lloesche** [reply]
So if you check the config https://github.com/valheimPlus/ValheimPlus/blob/master/valheim_plus.cfg
everything that is not in the `[Server]` section requires both the server and the client to have the mod. If you don't need any of those functions then the server only mod is fine but then `enforceMod` must be turned off.

**Reply by lloesche** [reply]
Example of one function I find very useful is `shareMapProgression=true` in the `[Map]` section. This requires both the server and client to have the mod.

**Reply by mryetin** [reply]
The fix is installing Valheim Plus on client side:
https://survivalservers.com/wiki/index.php?title=How_to_Install_Valheim_Plus

Thank you for helping!
